bin/check-examples.sh

```diff
@@ -8,64 +8,65 @@
 set -e
 
 THIS_DIR=$(dirname $0)
-
-for f in examples/jsonld/*.json; do
-    echo "Checking $f"
-
-    check-jsonschema \
-        -v \
-        --schemafile https://spdx.org/schema/3.0.0/spdx-json-schema.json \
-        $f
-
-    pyshacl \
-        -s https://spdx.org/rdf/3.0.0/spdx-model.ttl \
-        -e https://spdx.org/rdf/3.0.0/spdx-model.ttl \
-        $f
-done
-
-T=$(mktemp -d)
+SCHEMA_URL="https://spdx.org/schema/3.0.1/spdx-json-schema.json"
```

Comment:
I think it would be cleaner to write:
```suggestion
SPDX_VERSION="3.0.1"
SCHEMA_URL="https://spdx.org/schema/${SPDX_VERSION}/spdx-json-schema.json"
```